src/vterm/encoding/DECdrawing.tbl

```diff
@@ -0,0 +1,31 @@
+6/0 = U+25C6 # BLACK DIAMOND
```

Comment:
Question: should we vendor the tables as well? Or just move the comments to the include files?

(Longterm we probably want to replace all of this with `utf8proc`?)